scripts/travis/before_build.sh

```diff
@@ -11,8 +11,11 @@ fi
 
 npm install -g grunt-cli
 
-if [ $JOB != "ci-checks" ]; then
-  grunt package
+if [ $JOB != "ci-checks"]; then
+  if [ $JOB != "unit" ]; then
+    # unit tests test against unminified code
```

Comment:
I'm just trying some optimizations. Unit tests don't need package, and that shaves off a good minute of the unit tests. Not terribly important since e2e tests take longer anyway, but nice to have if you just want to know the unit tests pass.